presto-main/src/main/java/io/prestosql/server/security/jwt/JwtAuthenticatorSupportModule.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.server.security.jwt;
+
+import com.google.inject.Binder;
+import com.google.inject.Module;
+import com.google.inject.Scopes;
+import io.airlift.configuration.AbstractConfigurationAwareModule;
+import io.jsonwebtoken.SigningKeyResolver;
+
+import static io.airlift.configuration.ConditionalModule.installModuleIf;
+import static io.airlift.configuration.ConfigBinder.configBinder;
+import static io.airlift.http.client.HttpClientBinder.httpClientBinder;
+
+public class JwtAuthenticatorSupportModule
+        extends AbstractConfigurationAwareModule
+{
+    @Override
+    protected void setup(Binder binder)
+    {
+        configBinder(binder).bindConfig(JwtAuthenticatorConfig.class);
+        install(installModuleIf(
+                JwtAuthenticatorConfig.class,
+                JwtAuthenticatorSupportModule::isHttp,
+                new JwkModule()));
+        install(installModuleIf(
```

Comment:
`installModuleIf` takes otherwise parameter

Reply:
Thanks!  I didn't know about that.